EIPS/eip-9999.md

```diff
@@ -0,0 +1,285 @@
+---
+eip: 9999
+title: Enshrined Proposer Builder Separation
+description: Separates the ethereum block in consensus and execution parts, adds a mechanism for the consensus proposer to chose the execution proposer.
+author: Francesco D'Amato <francesco.damato@ethereum.org>, Barnabé Monnot <barnabe.monnot@ethereum.org>, Michael Neuder <michael.neuder@ethereum.org>, Potuz (@potuz), Terence Tsao <ttsao@offchainlabs.com>
```

Comment:
Not sure if it's intended, @potuz you forgot your email

Reply:
don't @ him ;P

Reply:
Need at least one github address to be the owner of the branch